templates/nextflow-aurora-mysql.yaml

```diff
@@ -326,9 +365,11 @@ Resources:
               secret = json.loads(secret_response['SecretString'])
               username = secret['username']
               password = secret['password']
+              log.debug(f'submit to database: {environ["DB_NAME"]}')
+              log.debug(f'using resourceArn: {environ["CLUSTER_ARN"]}')
               alter_db_response = submit_sql(f"ALTER DATABASE {db_name} CHARACTER SET utf8 COLLATE utf8_bin;")
               log.debug(f"Alter DB response: {alter_db_response}")
-              create_user_response = submit_sql(f"CREATE USER '{username}' IDENTIFIED BY '{password}';")
+              create_user_response = submit_sql(f"CREATE USER IF NOT EXISTS '{username}' IDENTIFIED BY '{password}';")
```

Comment:
When deployed, this event would occur multiple times and cause this statement to fail because the user might already have been created with a previous event.  AWS says that it's common and [users need to handle that by making their lambdas idempotent](https://repost.aws/knowledge-center/lambda-function-idempotent).  This change makes it work if we get duplicate events on deployment.